web-client/src/presenter/computeds/Public/publicCaseDetailHelper.test.js

```diff
@@ -366,6 +366,28 @@ describe('publicCaseDetailHelper', () => {
           isFileAttached: false,
           isLegacyServed: true,
         },
+        {
```

Comment:
This is a pretty large test already. Would it be possible to pull these expectations out into a test that is named more specifically for this new functionality?

Reply:
@rachelschneiderman, sounds good. Just pushed that change.

Reply:
Lovely! Thanks so much!